base/grisu.jl

```diff
@@ -13,9 +13,12 @@ include("grisu/float.jl")
 include("grisu/fastshortest.jl")
 include("grisu/fastprecision.jl")
 include("grisu/fastfixed.jl")
+include("grisu/bignums.jl")
 include("grisu/bignum.jl")
 
-function grisu(v::FloatingPoint,mode,requested_digits,buffer=DIGITS)
+const BIGNUMS = [Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum()]
```

Comment:
I wouldn't worry about it – we're going to need to deal with so many things that aren't yet threadsafe before that, this is hardly the end of the world.